src/test/spec/unified_runner/operation.rs

```diff
@@ -92,6 +95,22 @@ impl<'de> Deserialize<'de> for OperationObject {
     }
 }
 
+#[derive(Debug)]
+pub enum Expectation {
```

Comment:
With the new `ignoreResultAndError` case it seemed worthwhile to capture the valid states as an enum rather than continuing to add optional fields.